resources/metadata/gctsCreateRepository.yaml

```
@@ -0,0 +1,100 @@
+metadata:
+  name: gctsCreateRepository
+  description: Creates a Git repository on an ABAP system
+  longDescription: |
+    Creates a local Git repository on an ABAP system if it does not already exist.
+
+spec:
+  inputs:
+    secrets:
+      - name: credentialsId
+        description: Jenkins credentials ID containing username and password for authentication to the ABAP system on which you want to create the repository
+        type: jenkins
+    params:
+      - name: username
+        type: string
+        description: Username to authenticate to the ABAP system
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        aliases:
+          - name: username
+      - name: password
+        type: string
+        description: Password to authenticate to the ABAP system
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        aliases:
+          - name: password
+      - name: repositoryName
+        type: string
+        description: Specifies the name (ID) of the local repository
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        aliases:
+          - name: repositoryName
+      - name: host
+        type: string
+        description: Specifies the host address of the ABAP system including the port
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        aliases:
+          - name: host
+      - name: client
+        type: string
+        description: Specifies the client of the ABAP system to be adressed
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        aliases:
+          - name: client
+      - name: githubURL
```

I would suggest to rename this to reflect the repository scope.
```suggestion
      - name: remoteRepositoryURL
```